### PR TITLE
Add note about inertia being required for apply_torque on various Node types

### LINKED ISSUE #68068: apply_torque on RigidBody2D does not work if bodys CollisionShape2D is disabled
### Godot version

v4.0.beta3.official [01ae26d31]

### System information

Linux (Pop OS), NVIDIA GeForce RTX 2060, Video Drive v 470.141.03

### Issue description

If the CollisionShape2D under a RigidBody2D is disabled, using `apply_torque` in `_integrate_forces` or `_physics_process` does not appear to work.  

`apply_force` still works fine, which is what leads me to think this is a bug.

### Steps to reproduce

- Have` a `RigidBody2D` with a `CollisionShape2D`
- Have the ability to move straight (via `apply_force`), and rotate (via `apply_torque`)
- Disable the `CollisionShape2D`
- Notice that the `apply_force` call still works, but `apply_torque` does not.

In the example repro project:
- `w` thrusts
- `a` turns left
- `d` turns right
- `c` disables collision

I set it up to show debug collision shapes to better see what state the RigidBody2D is in.

### Minimal reproduction project

[RigidBodyRotateBugRepro.zip](https://github.com/godotengine/godot/files/9897214/RigidBodyRotateBugRepro.zip)

### PR DESCRIPTION
**Pull request by compmstr**
see: https://github.com/godotengine/godot/issues/68068

*Bugsquad edit:*
- Fixes #68068

### CODE REVIEW

**Comment by Calinou** [comment]
The same note is *likely* relevant for RigidBody3D and PhysicsDirectBodyState3D. Can you check there?

**Comment by compmstr** [comment]
Checked in the code, and calculating angular velocity uses _inv_inertia_tensor, so it appears that inertia is required for 3D as well.  Added a similar note to apply_torque functions for RigidBody3D

**Comment by mhilbrunner** [comment]
Please squash the commits into one :)

**Comment by compmstr** [comment]
I think that commit squashing is done upon merging?  Or am I missing an option in Github?

**Comment by Zireael07** [comment]
@compmstr: It can be done upon merging, but Godot engine repo doesn't use this option

**Comment by compmstr** [comment]
I followed https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase, and I think I've got it set up correctly.  TIL about manually squashing commits, thank you :)

I also added the note to PhysicsDirectBodyState3D.

**Review by mhilbrunner — APPROVED** [review]

**Comment by compmstr** [comment]
I fixed the CI issues:
* had to change 'param' to 'member' for the inertia members
* I had the class reference tags wrong (it's just `[ClassName]` instead of `[class ClassName]`)

File: doc/classes/PhysicsDirectBodyState2D.xml [hunk 1]
```diff
@@ -76,6 +76,7 @@
 			<param index="0" name="torque" type="float" />
 			<description>
 				Applies a rotational force without affecting position. A force is time dependent and meant to be applied every physics update.
+				[b]Note:[/b] [member inverse_inertia] is required for this to work.  To have [member inverse_inertia], an active [CollisionShape2D] must be a child of the node, or you can manually set [member inverse_inertia].
```
Comment (akien-mga) [inline doc/classes/PhysicsDirectBodyState2D.xml:79]: We only use one space after a period in the docs. (Same for all added sentences.)

**Comment by akien-mga** [comment]
Note that the email associated with your Git identity for this commit is not valid (see https://github.com/godotengine/godot/pull/68098.patch). It's not a problem per se but it means the commit can't be attributed to your GitHub account. If you want to fix that, you should change your Git identity locally and amend the commit with `git commit --amend --reset-author`.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git

**Comment by compmstr** [comment]
Thanks for the note about the email address.  I fixed that, and the double spaces after periods.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**